File: frontend/src/elements/Tags/StockTag.js
import React from "react";

const StockTag = ({ quantity }) => {
  return (
    <div className="PriceTag flex justify-center items-center bg-green-500 px-1 rounded">
      <h3 className="text-white font-semibold text-sm">{quantity} left</h3>
    </div>
  );
};

export default StockTag;

File: src/src/components/Menu/stock_tag.py
import reflex as rx

def StockTag(stock) -> rx.Component:
    return rx.chakra.box(
        rx.chakra.text(f"{stock} left", class_name="text-white font-semibold text-sm"),
        class_name="PriceTag flex justify-center items-center bg-green-500 px-1 rounded"
    )